oojs.js

```diff
@@ -3,10 +3,28 @@
   // A songs property that is determined by some input. It should contain multiple songs, each of which can just be a song title.
   // A favorites property that is initialized at 0. This cannot be set by user input.
   // An addSong method that adds a song to the songs property.
+class Playlist{
+  constructor(title){
+    this.title = title;
+    var favorites = 0;
+  }
+  addSong(){
+    const songs = [];
+    songs.push(this.title);
+    console.log(songs);
+  }
+}
 
 // Create an instance of the Playlist class.
+const beatle1 = new Playlist("Yesterday");
 
 // Create an Album class that inherits from Playlist. It should also...
   // Have an artist property that is determined by some input.
+class Album extends Playlist{
+  constructor(artist){
+    this.artist = artist;
```

Comment:
you have to have `super()` here as well--this calls the parent class' constructor function